aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -261,7 +269,7 @@ void PackedConvWeightCudnn<kSpatialDim>::apply_impl_helper(const at::Tensor& qua
     data_ptrs.reserve(10);
     uids.reserve(10);
     data_ptrs = {reinterpret_cast<int8_t*>(input.data_ptr()), conv_output.data_ptr(),
-                                           reinterpret_cast<int8_t*>(weight.data_ptr()),
+                                           reinterpret_cast<int8_t*>(orig_weight_.data_ptr()),
```

Comment:
should this happen in a different PR?

Reply:
I think something got messed up when I rebased yesterday. This should've already been done in a previous PR